Assets/Aircraft Physics/Core/Scripts/AircraftPhysics.cs

```diff
@@ -67,6 +72,12 @@ private void FixedUpdate()
         rb.AddTorque(currentForceAndTorque.q);
 
         rb.AddForce(transform.forward * thrust * thrustPercent);
+          if(isStopped)
+        {
+        rb.AddForce(-2*(transform.forward * thrust * thrustPercent));
```

Comment:
I know it is a bit hard to stop the plane but I've checked with real life statistics, the landing distance for the plane is about accurate (the braking amount is a bit high on my plane but other than that its fairly accurate). As far as I can tell this just adds a negative force, which is strange, because it is proportional to thrustpercent, which is being added in the lines above. The main technique for landing the plane is using flaps to add lift, slowing down, then touching down once the speech is already low enough to use brakes.